vm/actor/src/builtin/verifreg/mod.rs

```diff
@@ -0,0 +1,302 @@
+// Copyright 2020 ChainSafe Systems
+// SPDX-License-Identifier: Apache-2.0, MIT
+
+mod state;
+mod types;
+pub use self::state::State;
+pub use self::types::*;
+use crate::builtin::singletons::STORAGE_MARKET_ACTOR_ADDR;
+use crate::{HAMT_BIT_WIDTH, SYSTEM_ACTOR_ADDR};
+use address::Address;
+use ipld_blockstore::BlockStore;
+use ipld_hamt::BytesKey;
+use ipld_hamt::Hamt;
+use message::Message;
+use num_derive::FromPrimitive;
+use num_traits::Zero;
+use runtime::Runtime;
+use vm::{ActorError, ExitCode, METHOD_CONSTRUCTOR};
+/// Account actor methods available
+#[derive(FromPrimitive)]
+#[repr(u64)]
+pub enum Method {
+    Constructor = METHOD_CONSTRUCTOR,
+    AddVerifier = 2,
+    RemoveVerifier = 3,
+    AddVerifiedClient = 4,
+    UseBytes = 5,
+    RestoreBytes = 6,
+}
+
+pub struct Actor;
```

Comment:
need to impl `ActorCode` for this at the bottom of the file, once this is done, you can hook it up to be dispatched from the vm as the other actors are (ping me if you want clarification)